pkgs/development/python-modules/lexilang/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pytestCheckHook
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "LexiLang";
```

```suggestion
  pname = "lexilang";
```

We normalize the pname in python packages.